flake.lock

```diff
@@ -18,65 +18,54 @@
         "type": "github"
       }
     },
-    "nil-crypto3": {
+    "flake-utils_2": {
```

Comment:
why we still need "flake-utils_2"? one version of the "flake-utils" not enough?
same about "system-2" and "nix-3rdparty_2"

Reply:
That happens due to packages duplication.
I have corrected it now: flake-utils and 3rd-party are now in sync with crypto3 versions